### ISSUE TITLE
retainning walls with adapted relief?

### ISSUE THREAD

**Issue by Matthias84** [StrandedKitty/streets-gl#32]
There are retaining walls, that realise a hard height shift in reality. Usually they hold / keep away ground from a higher level from a lower level. Would be nice to see this objects, but yes they affect your DEM as well.

**Comment by rskedgell** [comment]
Would it be worth considering this only where height=* or est_height=* is set? We have barrier=retaining_wall for everything between the surrounds of 1 metre high raised beds in accessible gardens to 10 metre high bridge and tunnel approaches on motorways.

**Comment by StrandedKitty** [comment]
I'm not sure it's possible to accurately render retaining walls. The terrain is just not detailed enough for this.

**Comment by Matthias84** [comment]
Yeah and I guess enhancing the DTM is also limited by the internal representation? (raster vs. TIN).
And AFAIK ist this topic also challenging for other devs e.g. O2W https://wiki.openstreetmap.org/wiki/OSM2World#Elevation_data @tordanik describes some nice approaches and tricks in his talks, but it's a real challenge to find generic solutions that don't cause to hard artifacts on other places https://media.ccc.de/search/?q=OSM2world
